source/runtime_gui.cpp

```diff
@@ -1108,6 +1109,13 @@ void reshade::runtime::draw_ui_settings()
 			_tutorial_index = 0;
 	}
 
+	if (ImGui::CollapsingHeader("Effects & Techniques", ImGuiTreeNodeFlags_DefaultOpen))
```

Comment:
I'd move this in the `General` section, since the relevant settings are stored in the `GENERAL` section of the INI and I have started to try and map INI sections to sections on the Settings page more (hence why there is now a `OVERLAY` and `SCREENSHOTS` section in the INI).
Even better yet would be to move the `Show \"Force load all effects\" button` into the `Overlay` section while at it.

Reply:
![loads_only_enabled_effects2](https://user-images.githubusercontent.com/71371622/93711567-5c405500-fb8a-11ea-8a3e-ddc832e3a723.png)
I do not like this. :thinking:

Reply:
Gotta say I do :stuck_out_tongue:. The "Load only enabled effects" checkbox is now closer to the effect search path setting, which is a nice corelation and IMHO it makes sense to have any settings that affect the overlay to be in the overlay section (similar to the "Group effect files ..." one).

Reply:
Sorry I can't found good position.
If another changes has no problem, I recommend to change this by yourself. 🙏